internal/backends/sqlite/collection.go

```diff
@@ -59,6 +63,12 @@ func (c *collection) Query(ctx context.Context, params *backends.QueryParams) (*
 
 	rows, err := db.QueryContext(ctx, query)
 	if err != nil {
+		// No such table, return empty result.
```

Comment:
That should be documented for `collectionContract.Query`

Reply:
Added a comment.